frontend/src/metabase/query_builder/components/expressions/ExpressionEditorHelpText/ExpressionEditorHelpText.module.css

```diff
@@ -0,0 +1,60 @@
+.ArgumentsGrid {
+  margin: 1rem 0;
```

Comment:
:x: Arguments are now `rgb(227, 89, 94)` but used to be `rgb(239, 140, 140)`

![image](https://github.com/user-attachments/assets/111a8b63-214b-4f4d-ba9c-271af80f5fd5)

Reply:
it's fine, we agreed to use this color https://metaboat.slack.com/archives/C0645JP1W81/p1733315783925849?thread_ts=1733146244.214589&cid=C0645JP1W81